.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+dist
 .DS_Store
 *.pem
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
was you meant to ignore package-lock ?

Reply:
yes package lock is not needed, the script is going to be build and upload on a cdn